benchmarks/tpcc/src/main/java/com/google/cloud/pgadapter/tpcc/dataloader/DataLoader.java

```diff
@@ -52,6 +63,7 @@ public class DataLoader implements AutoCloseable {
   public DataLoader(
       DataLoadStatus status, String connectionUrl, TpccConfiguration tpccConfiguration) {
     this.connectionUrl = connectionUrl;
+    this.isGoogleSQL = connectionUrl.startsWith("jdbc:cloudspanner");
```

Comment:
Here also: I think this gives the wrong result if you are running a benchmark for a PostgreSQL-dialect database using the Spanner JDBC driver.

Reply:
Updated as well.